<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>MS Sans Serif</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>400</width>
     <height>400</height>
    </rect>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>311</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 247, 209);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-image: url(:/Button/UI/Element/plus.png);
	background-position: center; 
	background-repeat: no-repeat;
	border: 0;
}
QPushButton:hover{
	background-position: 10px 0px;
}
QPushButton:pressed{
	background-position: center;
	border:1;
	border-style:inset;
	border-color: rgb(212, 212, 212);
	
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="controlTools" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>330</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="boldButton">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-image: url(:/Button/UI/Element/bold.png);
	background-position: center; 
	background-repeat: no-repeat;
	border: 0;
}
QPushButton:hover{
	background-position: bottom right;
}
QPushButton:pressed{
	background-position: center;
	border:1;
	border-style:inset;
	border-color: rgb(212, 212, 212);
	
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="italicButton">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-image: url(:/Button/UI/Element/italic.png);
	background-position: center; 
	background-repeat: no-repeat;
	border: 0;
}
QPushButton:hover{
	background-position: bottom right;
}
QPushButton:pressed{
	background-position: center;
	border:1;
	border-style:inset;
	border-color: rgb(212, 212, 212);
	
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="underlineButton">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-image: url(:/Button/UI/Element/underline.png);
	background-position: center; 
	background-repeat: no-repeat;
	border: 0;
}
QPushButton:hover{
	background-position: bottom right;
}
QPushButton:pressed{
	background-position: center;
	border:1;
	border-style:inset;
	border-color: rgb(212, 212, 212);
	
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="strikethroughButton">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-image: url(:/Button/UI/Element/strikethrough.png);
	background-position: center; 
	background-repeat: no-repeat;
	border: 0;
}
QPushButton:hover{
	background-position: bottom right;
}
QPushButton:pressed{
	background-position: center;
	border:1;
	border-style:inset;
	border-color: rgb(212, 212, 212);
	
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="listButton">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-image: url(:/Button/UI/Element/TB.png);
	background-position: center; 
	background-repeat: no-repeat;
	border: 0;
}
QPushButton:hover{
	background-position: bottom right;
}
QPushButton:pressed{
	background-position: center;
	border:1;
	border-style:inset;
	border-color: rgb(212, 212, 212);
	
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cryptoButton">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-image: url(:/Button/UI/Element/unlocked.png);
	background-position: center; 
	background-repeat: no-repeat;
	border: 0;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="topicTools" native="true">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="colorButton">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-image: url(:/Button/UI/Element/other.png);
	background-position: center; 
	background-repeat: no-repeat;
	border: 0;
}
QPushButton:hover{
	background-position: 10px 0px;
}
QPushButton:pressed{
	background-position: center;
	border:1;
	border-style:inset;
	border-color: rgb(212, 212, 212);
	
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-image: url(:/Button/UI/Element/x.png);
	background-position: center; 
	background-repeat: no-repeat;
	border: 0;
}
QPushButton:hover{
	background-position: 10px 0px;
}
QPushButton:pressed{
	background-position: center;
	border:1;
	border-style:inset;
	border-color: rgb(212, 212, 212);
	
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>311</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0.795, y1:0.193182, x2:0.136182, y2:0.858, stop:0.125 rgba(58, 58, 58, 182), stop:1 rgba(108, 108, 108, 72));
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="invisiableArea">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>330</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>271</width>
      <height>251</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Serif</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>label</zorder>
   <zorder>addButton</zorder>
   <zorder>topicTools</zorder>
   <zorder>invisiableArea</zorder>
   <zorder>controlTools</zorder>
   <zorder>textEdit</zorder>
  </widget>
 </widget>
 <resources>
  <include location="../UIresource.qrc"/>
 </resources>
 <connections/>
</ui>
